.github/workflows/node.js.yml

```diff
@@ -28,3 +28,20 @@ jobs:
     - run: npm install
     - run: npm run build --if-present
     - run: npm test
+
+  build-and-deploy:
+    
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - run: npm install
+    - run: npm run build
+    - name: Remove .gitignore
```

Comment:
Question: Why do we need to remove `gitignore`?

Reply:
because we need the `dist` folder in the dmoe page, the next step of `deploy-action` will ignore the file/folder that list under `.gitignore`

Reply:
Can we add that as a comment on this step?

Reply:
make sense 👍 done.